### PR TITLE
(feat) Ah docs

### PR DESCRIPTION
**Pull request by austbot**
- [x] Faq
- [x] link to mpl
- [x] basic docs
- [x] roadmap
- [ ] js example
- [ ] program api
- [x] cli
- [x] overview

### CODE REVIEW

File: docs/candy-machine-v1/01-introduction.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 :::warning
 Candy Machine v1 has been deprecated. Creating a new instance of Candy Machine v1 is no longer possible. Please use [Candy Machine v2](../candy-machine-v2/introduction) instead.
-:::
```
Comment (stegaBOB) [inline docs/candy-machine-v1/01-introduction.md:5]: Add back the warning
```suggestion
:::

```

File: docs/auction-house/definition.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: "What is AuctionHouse"
+sidebar_position: 1
+---
+
+# What is Auction House
+
+AuctionHouse is a protocol for marketplaces to implement a decentralized sales contract. It is simple, fast and very cheap. AuctionHouse is a Solana Program available on mainnet and devnet. Anyone can create an AuctionHouse and accept any SPL token they wish.
```
Comment (stegaBOB) [inline docs/auction-house/definition.md:8]:
```suggestion
AuctionHouse is a protocol for marketplaces to implement a decentralized sales contract. It is simple, fast and very cheap. AuctionHouse is a Solana Program available on mainnet-beta and devnet. Anyone can create an AuctionHouse and accept any SPL token they wish. 
```

File: docs/auction-house/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: "Getting Started"
+sidebar_position: 2
+---
+# Getting Started with AuctionHouse
+
+The AuctionHouse is a Solana program running on Mainnet and Devnet. While you may interact with it like any other Solana program by sending transactions to a Solana node, Metaplex has built some tools to make working with it much easier. We have a CLI tool that will let you manage your auction house and some reference JS code to help you build a user interface.
```
Comment (stegaBOB) [inline docs/auction-house/getting_started.md:7]:
```suggestion
The AuctionHouse is a Solana program running on Mainnet Beta and Devnet. While you may interact with it like any other Solana program by sending transactions to a Solana node, Metaplex has built some tools to make working with it much easier. We have a CLI tool that will let you manage your auction house and some reference JS code to help you build a user interface. 
```

**Review by stegaBOB — CHANGES_REQUESTED** [review]
One error fix and two mainnet -> mainnet-beta nits

**Review by stegaBOB — APPROVED** [review]

**Comment by roederw** [comment]
I still need to review (but I can always push fixes up afterwards). Go ahead and merge!

**Pull request merged**